src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -401,6 +407,30 @@ private void AddCredentialToCache(Uri serverUri, ulong authAvail, NetworkCredent
             }
         }
 
+        private void AddResponseCookies(Uri serverUri, HttpResponseMessage response)
+        {
+            if (!_useCookie)
```

Comment:
@stephentoub , please see the constructor of [HttpClientHandler.Windows.cs](https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Windows.cs#L145). HttpClientHandler on windows has UseCookies set to True as default. As @davidsh  [kindly pointed out](https://github.com/dotnet/corefx/issues/3016#issuecomment-136487885) in #3016 there are tests that validate this property too.